apps/condo/migrations/20221111002750-0186_auto_20221110_1927.js

```diff
@@ -0,0 +1,34 @@
+// auto generated by kmigrator
+// KMIGRATOR:0186_auto_20221110_1927:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDMuMi43IG9uIDIwMjItMTEtMTAgMTk6MjcKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAxODVfd2ViaG9va3N1YnNjcmlwdGlvbl9mYWlsdXJlc2NvdW50X2FuZF9tb3JlJyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSdyZW1vdGVjbGllbnQnLAogICAgICAgICAgICBuYW1lPSdwdXNoVHlwZScsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5DaGFyRmllbGQoYmxhbms9VHJ1ZSwgY2hvaWNlcz1bKCdkZWZhdWx0JywgJ2RlZmF1bHQnKSwgKCdzaWxlbnRfZGF0YScsICdzaWxlbnRfZGF0YScpXSwgbWF4X2xlbmd0aD01MCwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3JlbW90ZWNsaWVudGhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdwdXNoVHlwZScsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5UZXh0RmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgXQo=
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Add field pushType to remoteclient
+--
+ALTER TABLE "RemoteClient" ADD COLUMN "pushType" varchar(50) NULL;
```

Comment:
why NULL ? we have a default value. and probably we need to add a not null constraint